### PR TITLE
Katya

### PR DESCRIPTION
**Pull request by katerynabebeshko**
Homework 1, 2, 3

### CODE REVIEW

File: KatyaBebeshko/HW1/task1.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+userName = input('What is your name?: ')
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW1/task1.py:1]: good job

File: KatyaBebeshko/HW1/task2.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+a = 789
+b = 367
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW1/task2.py:3]: good job
try to use f string for output information like this:
print(f"a+b={a+b}") or print(f"sum={a+b}")

File: KatyaBebeshko/HW3/task1.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+49.In the face of ambiguity, refuse the temptation to guess.
+50.There should be one-- and preferably only one --obvious way to do it.
+51.Although that way may not be obvious at first unless you're Dutch.
+52.Now is better than never.
+53.Although never is often better than *right* now.
+54.If the implementation is hard to explain, it's a bad idea.
+55.If the implementation is easy to explain, it may be a good idea.
+56.Namespaces are one honking great idea -- let's do more of those!'''
+
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW3/task1.py:60]: good job

File: KatyaBebeshko/HW3/task2.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# SUBTASK2
+## find the product of the digits of this number
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW3/task2.py:2]: good job

File: KatyaBebeshko/HW3/task3.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# SUBTASK3
+## Interchange the values of two variables without using the third variable.
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW3/task3.py:2]: good job
another way:
a, b = b, a

File: KatyaBebeshko/HW4/practice_tasks.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Task2. Write a script, which of the two entered numbers will determine which of them is more
+# and which is less. Take into account the case of equality of numbers.
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW4/practice_tasks.py:2]: good job

File: KatyaBebeshko/HW4/task1.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+# Task1. Write a script that will calculate the factorial
+# of the entered number without using recursion.
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW4/task1.py:2]: good job

File: KatyaBebeshko/HW5/practice_tasks.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
+# Print all even numbers less than 100 (write two variants of the code:
+# one using the while loop, and the other using the for loop).
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW5/practice_tasks.py:3]: good job

File: KatyaBebeshko/HW5/task2.py [hunk 9]
```diff
@@ -0,0 +1,15 @@
+# Task2. Create a list that contains elements of integer type,
+# then use the loop to change the type of these elements to a floating type.
+# (Hint: use the built-in float () function).
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW5/task2.py:4]: good job

File: KatyaBebeshko/HW5/task3.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
+# Task3. Print Fibonacci numbers up to the entered number n, using cycles.
+# (Sequence of Fibonacci numbers 0, 1, 1, 2, 3, 5, 8, 13, etc.)
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW5/task3.py:3]: another way of implementation code:
n = int(input('enter some number from fibonacci list='))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')

File: KatyaBebeshko/HW6/practice_task.py [hunk 11]
```diff
@@ -0,0 +1,11 @@
+# Create a list of integers that are entered from the terminal 
+# and determine the maximum and minimum number among them.
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW6/practice_task.py:3]: good

File: KatyaBebeshko/HW6/task1.py [hunk 12]
```diff
@@ -0,0 +1,16 @@
+# Task1. In the range from 1 to 10 determine 
+# • even numbers that are divisible by 2,
+# • odd numbers, which are divisible by 3,
+# • numbers that are not divisible by 2 and 3.
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW6/task1.py:5]: good job

File: KatyaBebeshko/HW6/task2.py [hunk 13]
```diff
@@ -0,0 +1,18 @@
+# Task2. Write a script that checks the login that the user enters.
+# If the login is "First", then greet the users. If the login is different, send an error message.
+# (need to use loop while)
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW6/task2.py:4]: good job
but the task condition did not predict the number of trials

Comment (katerynabebeshko) [reply KatyaBebeshko/HW6/task2.py:4]: agree

here is a correct way

while input('Enter login: ') != 'First':
    print('Access denied. Try again.')
else:
    print('Successful login')

File: KatyaBebeshko/HW7/task3.py [hunk 14]
```diff
@@ -0,0 +1,16 @@
+# Task3. Write a function that calculates the number of characters
+# included in a given string
+# • input: "hello"
+# • output: {"h":1, "e":1,"l":2,"o":1}
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW7/task3.py:5]: good job

File: KatyaBebeshko/HW8/calculator.py [hunk 15]
```diff
@@ -0,0 +1,11 @@
+def addition(num1, num2):
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW8/calculator.py:1]: good job

File: KatyaBebeshko/HW8/task2.py [hunk 16]
```diff
@@ -0,0 +1,28 @@
+# Task2.
+# Write a Python program to check the validity of a password (input from users).
+# Validation :
+# At least 1 letter between [a-z] and 1 letter between [A-Z]. At least 1 number between [0-9].
+# At least 1 character from [$#@].
+# Minimum length 6 characters.
+# Maximum length 16 characters.
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW8/task2.py:8]: good job

File: KatyaBebeshko/HW8/task3_area_functions.py [hunk 17]
```diff
@@ -0,0 +1,22 @@
+# Task3.
+# Write a program that calculates the area of a rectangle S=a*b, the area of a
+# triangle S=0.5*h*a, the area of a circle S=pi*r**2. This module must be used in
+# another module in which we ask the user the area of which figure he wants to
+# calculate.
+# (To perform the task, you need to import the math module, and from it the
+# pow() function and the value of the variable pi, and module, which contains
+# three functions for finding areas, into the main program. The basic logic of the
+# program is executed in the main module).
+
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW8/task3_area_functions.py:10]: good job

File: KatyaBebeshko/HW8/task3_ask_user.py [hunk 18]
```diff
@@ -0,0 +1,15 @@
+import task3_area_functions as areas
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW8/task3_ask_user.py:1]: good job

File: KatyaBebeshko/HW8/to_use_calc.py [hunk 19]
```diff
@@ -0,0 +1,18 @@
+import calculator as calc
```
Comment (kolyasalubov) [inline KatyaBebeshko/HW8/to_use_calc.py:1]: good job

**Pull request merged**